indigo/distribution.yaml

```diff
@@ -6476,6 +6476,12 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: indigo-devel
     status: maintained
+  peter_task_action:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: ssh://zhangwei@yunjiapp.com:29418/zhangwei/test_release.git
```

Comment:
I think the release repo must even be on github.com (as the error message in the travis log indicates).